parser.y

```diff
@@ -1449,9 +1450,9 @@ PlacementOpts:
 	}
 
 PlacementSpec:
-	"ADD" "PLACEMENT" PlacementOpts
+	"ADD" "PLACEMENT" "POLICY" PlacementOpts
```

Comment:
Could we remove `POLICY`? I think `ADD PLACEMENT CONSTRAINTS...` is OK. I think this statement is relatively long, if not necessary, we can remove this field.

Reply:
@djshow832 what do you think? You have not updated the RFC doc yet... For me, both makes sense.

Reply:
You can discuss it under the issue https://github.com/pingcap/tidb/issues/18030 @zimulala

Reply:
Fine.